src/page-modules/departures/server/journey-planner/index.ts

```diff
@@ -230,6 +241,11 @@ export function createJourneyApi(
             return acc;
           }
 
+          const situations =
+            edge.node.place.quays
+              ?.map((q) => q?.situations ?? ([] as GraphQlSituation[]))
+              ?.flat() ?? [];
```

Comment:
Is this the same as doing `flatMap()`?

Reply:
Yes, this could be refactored to use flatMap